### PR TITLE
fix issue 1504 & 1854, modified discovery messages

### PR DESCRIPTION
**Pull request by xuweibj**
#1504 #1854

### CODE REVIEW

File: xCAT-server/lib/xcat/plugins/typemtms.pm [hunk 1]
```diff
@@ -36,14 +36,14 @@ sub findme {
...
+        xCAT::MsgUtils->message("S", "xcat.discovery.mtms: ($request->{_xcat_clientmac}->[0]) Warning: Could not find any node matching using mtms-based discovery");
         return;
     } elsif ($nodenum > 0) {
-        xCAT::MsgUtils->message("S", "xcat.discovery.mtms: ($request->{_xcat_clientmac}->[0]) Error: More than one node were found");
+        xCAT::MsgUtils->message("S", "xcat.discovery.mtms: ($request->{_xcat_clientmac}->[0]) Error: More than one node for $mtms were found");
         return;
     }
     {
-        xCAT::MsgUtils->message("S", "xcat.discovery.mtms: ($request->{_xcat_clientmac}->[0]) Found node: $nodes[0]");
+        xCAT::MsgUtils->message("S", "xcat.discovery.mtms: ($request->{_xcat_clientmac}->[0]) Found node: $nodes[0] matching for discovery request");
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/typemtms.pm]: I don't think this msg is necessary to change ( same comment for all messages changed in this format)

File: xCAT-server/lib/xcat/plugins/blade.pm [hunk 2]
```diff
@@ -4546,7 +4546,7 @@ sub process_request {
             }
         }
         unless ($node) {
-            xCAT::MsgUtils->message("S", "xcat.discovery.blade: ($request->{_xcat_clientmac}->[0]) Warning: Could not find any nodes using blade-based discovery");
+            xCAT::MsgUtils->message("S", "xcat.discovery.blade: ($request->{_xcat_clientmac}->[0]) Warning: Could not find any node matching using blade-based discovery");
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/blade.pm]: If we want to change this msg, suggest :  

`Could not find any predefined nodes in xCAT matching the discovered node (<fill in node name>) using XXX-based discovery.`

Applies to all the changes below with the same message format.

File: xCAT-server/lib/xcat/plugins/switch.pm [hunk 3]
```diff
@@ -318,7 +318,7 @@ sub process_request {
         }
 
         if ($node) {
-            xCAT::MsgUtils->message("S", "xcat.discovery.switch: ($req->{_xcat_clientmac}->[0]) Found node: $node");
+            xCAT::MsgUtils->message("S", "xcat.discovery.switch: ($req->{_xcat_clientmac}->[0]) Found node: $node matching for discovery request");
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/switch.pm]: I don't think this msg is necessary to change

File: xCAT-server/lib/xcat/plugins/seqdiscovery.pm [hunk 4]
```diff
@@ -384,7 +384,7 @@ sub findme {
 
         # call the discovered command to update the discovery request to a node
 
-        xCAT::MsgUtils->message("S", "xcat.discovery.seqdiscovery: ($request->{_xcat_clientmac}->[0]) Found node: $node");
+        xCAT::MsgUtils->message("S", "xcat.discovery.seqdiscovery: ($request->{_xcat_clientmac}->[0]) Found node: $node matching for discovery request");
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/seqdiscovery.pm]: I don't think this msg is necessary to change

File: xCAT-server/lib/xcat/plugins/nodediscover.pm [hunk 5]
```diff
@@ -463,7 +462,7 @@ sub process_request {
     #Update the discoverydata table to indicate the successful discovery
     xCAT::DiscoveryUtils->update_discovery_data($request);
 
-    syslog("local4|info", "$node has been discovered");
+    xCAT::MsgUtils->message("S", "xcat.discovery.nodediscover: $node has been discovered so we can correlate the discovered definition to the pre-defined definition.");
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/nodediscover.pm]: I don't think this message is necessary to be changed

File: xCAT-server/lib/xcat/plugins/blade.pm (hunk 2 above)
Comment (zet809) [inline xCAT-server/lib/xcat/plugins/blade.pm]: Only a MAC can be get here, so I suggest we won't modify this message.

File: xCAT-server/lib/xcat/plugins/typemtms.pm [hunk 6]
```diff
@@ -36,14 +36,14 @@ sub findme {
     }
     my $nodenum = $#nodes;
     if ($nodenum < 0) {
-        xCAT::MsgUtils->message("S", "xcat.discovery.mtms: ($request->{_xcat_clientmac}->[0]) Warning: Could not find any nodes using mtms-based discovery");
+        xCAT::MsgUtils->message("S", "xcat.discovery.mtms: ($request->{_xcat_clientmac}->[0]) Warning: Could not find any node matching using mtms-based discovery");
```
Comment (zet809) [inline xCAT-server/lib/xcat/plugins/typemtms.pm]: Changing to "Could not find any node matching $mtms using mtms-based discovery" will be more helpful.

**Comment by zet809** [comment]
Looks good to me, I agree to merge.

**Pull request merged**